erpnext/accounts/doctype/sales_invoice/sales_invoice.js

```diff
@@ -1113,6 +1113,14 @@ frappe.ui.form.on('Sales Invoice', {
 				}
 			}
 		};
+
+		frm.set_query("order_picker", function () {
+			return {
+				filters: {
+					status: "Enabled"
+				}
+			};
+		});
```

Comment:
Should be in `moghul_custom` app, not in `erpnext`. Order Picker does not exist in erpnext and is a company specific feature